adoc-mode.el

```diff
@@ -3002,6 +2993,38 @@ or \\[adoc-toggle-inline-images]."
           (adoc-create-image-overlay file start end)
           )))))
 
+(defun adoc-image-overlays (&optional begin end)
+  "Return list of image overlays in region from BEGIN to END.
+BEGIN and END default to the buffer boundaries
+ignoring any restrictions."
+  (save-restriction
+    (widen)
+    (unless begin (setq begin (point-min)))
+    (unless end (setq end (point-max)))
+    (let (image-overlays)
+      (dolist (ov (overlays-in begin end))
+        (when (overlay-get ov 'adoc-image)
+          (push ov image-overlays)))
+      image-overlays)))
+
+(defun adoc-remove-images ()
+  "Remove inline image overlays from image links in the buffer.
+This can be toggled with `adoc-toggle-images'
+or \\[adoc-toggle-images]."
+  (interactive)
+  (let ((image-list (adoc-image-overlays)))
+    (when image-list
+      (dolist (ov image-list)
+        (delete-overlay ov))
+      t)))
+
+(defun adoc-toggle-images ()
```

Comment:
I think this can be `adoc-toggle-inline-images` or `adoc-toggle-display-images`.

Reply:
This affects block images as well as inline images. Therefore I removed the "inline".

Reply:
What's a block image?

Reply:
https://docs.asciidoctor.org/asciidoc/latest/macros/images/

Reply:
Ah, you mean the AsciiDoc spec - for "inline" images are those that we display inline in Emacs. :-) Anyways, I guess we can avoid "inline" in the names, as long as they are consistent across all functions.

Reply:
Thanks for bringing up this topic. Otherwise clarification would be impossible.

In the source code of `adoc-mode.el` the word "inline" is consistently used in the sense of the Adoc-specification (as the opposite of "block").